docs/tutorials/first-steps.rst

```diff
@@ -29,6 +29,7 @@ We'll start with a relatively simple data model:
 
 Using the `httpie <http://httpie.org>` tool we can post a sample record in the
 ``tasks`` collection:
+For proxy issues we can use  "http --proxy=http:http://usrname:passwrd@host:port COMMAND" instead of "http COMMAND"
```

Comment:
I am sure there should be a way to define the PROXY in the ENV. See https://github.com/jkbrzt/httpie#proxies